vmdb/app/helpers/container_service_helper/textual_summary.rb

```diff
@@ -53,12 +52,16 @@ def textual_portal_ip
     {:label => "Portal IP", :value => @record.portal_ip}
   end
 
-  def textual_protocol
-    {:label => "Protocol", :value => @record.protocol}
-  end
 
-  def textual_container_port
-    {:label => "Container Port", :value => @record.container_port}
+  def textual_port_config(single_conf)
+    name = single_conf.name
+
+    name = "Unnamed" if name.nil? || name.empty?
+
+    {
+      :label => name,
+      :value => "#{single_conf.protocol} port:#{single_conf.port} to port:#{single_conf.target_port}"
```

Comment:
perhaps target_port? it's not ideal, but that's the field name, and it will look less like a port range.
or maybe add some verb there next to the "to"? 
worth to check the target port definition on k8s side, maybe there's something there that makes sense to use here?